.github/workflows/black.yml

```diff
@@ -7,4 +7,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3.0.2
-      - uses: psf/black@22.8.0
+      - uses: psf/black@stable
+        with:
+          version: "~= 22.0"
```

Comment:
We prefer to strictly pin

Reply:
pinned to "psf/black@22.8.0" hopefully it should respect stabilty `version` too

Reply:
Unfortunately it looks like `psf/black@22.8.0` does not support `version`. 

![image](https://github.com/home-assistant-libs/forecast_solar/assets/10961167/580efda0-4040-4aea-a54c-2e91e4cf8134)

Reply:
Looks like `psf@24.3.0` works

![image](https://github.com/home-assistant-libs/forecast_solar/assets/10961167/d79f6651-ff26-4df2-be90-84505a42a27d)